invite/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2020 The AMP HTML Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const {serverless} = require('@probot/serverless-gcf');
+const appFn = require('./app');
+module.exports.probot = serverless(appFn);
```

Comment:
This file defines the **serverless wrapper** around the Probot application, allowing it to be used in a serverless Cloud Functions deployment.